### DISCUSSION TITLE
The `TransparentPointer` Control

### DISCUSSION THREAD

**Discussion by ndonkoHenri** [flet-dev/flet#658] [Q&A] [answered]
I am just from seeing this control in the python sdk, but don't think it has already been mentioned in the docs,
I will eventually like to know more...

**Comment by FeodorFitsner** [comment, marked as answer]
So, it's a solution for the problem from here: https://stackoverflow.com/questions/65269190/pass-trough-all-gestures-between-two-widgets-in-stack

Imagine if you have an `ElevatedButton` inside `Container` with `GestureDetector` then tapping on a button won't be visible to a gesture detector behind it. With `TransparentPointer` a tapping event doesn't stop on a button, but goes up to the parent, kind of event bubbling in HTML/JS.

Look at the same usage here: https://github.com/flet-dev/flet/blob/main/studio/server/main.py#L49-L61

Whatever clickable controls you have inside `FletApp` container all their events are visible to gesture detector.

Would Flet community benefit from that control, so I have to document it?

**Reply by ndonkoHenri** [reply]
> Would Flet community benefit from that control, so I have to document it?

It might be useful someday. I think you should just document all those 'secret' controls you've made. 😉

May I ask why put `FletApp` in a `TransparentPointer`? (But there is no GestureDetector..¿)

By the way, is `FletApp` already ready? If yes, please document it with an example..

**Reply by FeodorFitsner** [reply]
Here you are: https://flet.dev/docs/controls/transparentpointer
